misc/python/materialize/mzcompose/services.py

```diff
@@ -46,8 +46,6 @@
     "storage_persist_sink_minimum_batch_updates": "100",
     "persist_pubsub_push_diff_enabled": "true",
     "persist_pubsub_client_enabled": "true",
-    "persist_stats_filter_enabled": "true",
```

Comment:
@philip-stoev Do you have any preference on, when we change the default for a new feature to "on", if we leave these here or clean them up?